komga-webui/src/locales/en.json

```diff
@@ -230,37 +230,37 @@
     "book_number": "Book number: {name}",
     "book_series": "Series: {name}",
     "button_import": "Import",
-    "comicrack_preambule_html": "You can import existing ComicRack Reading Lists in <code>.cbl</code> format.<br/>Komga will try to match the provided series and book number with series and books in your libraries.",
```

Comment:
This just opens up potential, instead of directing.

Reply:
"When import" doesn't seem right to me.